client/src/app/app.component.scss

```diff
@@ -0,0 +1,6 @@
+.flex-wrapper {
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
+    justify-content: space-between
```

Comment:
Thanks for the fix! 👍 
I was planning to focus on this issue. Btw. nice way how to fix it = "space between"